src/eatery_db/campus_eatery.py

```diff
@@ -68,7 +65,7 @@ def parse_campus_hours(data_json, eatery_model):
         eatery_slug = eatery.get("slug")
```

Comment:
should there be a default value (ex. `eatery.get("slug","")` ) in case there's an error?

Reply:
Added